pkg/controller/propagator/propagation.go

```diff
@@ -310,6 +324,16 @@ func (r *ReconcilePolicy) handleDecision(instance *policiesv1.Policy, decision a
 			// Make sure the Owner Reference is cleared
 			replicatedPlc.SetOwnerReferences(nil)
 
+			//do a quick check for any template delims in the policy before putting it through
+			// template processor
+			if policyHasTemplates(instance) {
+				//resolve hubTemplate before replicating
+				err = processTemplates(replicatedPlc, decision, instance.GetNamespace())
+				if err != nil {
+					return err
```

Comment:
I did bring this up in the scrum yesterday. That is the case with any error in  the  propagator  right now not just templates, any error in the propagation process is jsut being bubbled up instead of appropriately marking the status on the policy.

I can just log  and swallow, and take an issue to fix up the error handling in the propagator and but I do not intend to handle this in this PR

Reply:
As discussed earlier, let's swallow up the error and record an error event.